ui/lib/kubernetes/addon/components/page/role/details.hbs

```diff
@@ -0,0 +1,72 @@
+<PageHeader as |p|>
+  <p.top>
+    <nav class="breadcrumb" aria-label="breadcrumbs" data-test-breadcrumbs>
+      <ul>
+        <li data-test-crumb="overview">
+          <span class="sep">/</span>
+          <LinkTo @route="overview">{{@model.backend}}</LinkTo>
+        </li>
+        <li data-test-crumb="roles">
+          <span class="sep">/</span>
+          <LinkTo @route="roles">roles</LinkTo>
+        </li>
+        <li>
+          <span class="sep">/</span>
+          <span data-test-crumb="role">{{@model.name}}</span>
+        </li>
+      </ul>
+    </nav>
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-header-title>
+      {{@model.name}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<Toolbar>
+  <ToolbarActions>
+    <ConfirmAction @buttonClasses="toolbar-link" @onConfirmAction={{this.delete}} data-test-delete>
```

Comment:
Should there be capabilities checks for all of these actions? I like how Chelsea added them in her PR [here](https://github.com/hashicorp/vault/pull/18229/commits/ec768bf67907c661acb5c926d36f6ef2aa07af0b)

which reminds me I need to add them for the pki key details view 🤔

Reply:
Oh yes definitely. I had capabilities checks somewhere else in the project and totally forgot to add them here. Thanks for catching that!

Reply:
Just to confirm, the pattern is to hide the actions rather than disable if the user does not have the capability?